events/tests/test_importer.py

```diff
@@ -45,7 +44,7 @@ def test_modified_event(self):
 DESCRIPTION:<a href="https://www.barcamptools.eu/pycamp201604">PythonCamp C
  ologne 2016</a>
 LAST-MODIFIED:20160401T210533Z
-LOCATION:GFU Cyrus AG\, Am Grauen Stein 27\, 51105 Köln\, Germany
+LOCATION:GFU Cyrus AG, Am Grauen Stein 27, 51105 Köln, Germany
```

Comment:
Why don't we need the escape char anymore?

Reply:
Pycharm warned me about the error.

![image](https://user-images.githubusercontent.com/807599/128893274-ecaa4f7f-b7a3-443c-9a5f-f77e77fdfb28.png)

https://docs.python.org/3/whatsnew/3.6.html#deprecated-python-behavior

Triple double-quote string is already a raw string, so I don't think it would make sense to put an `r` on the string definition. (please correct me if my conclusion is wrong)